src/qml/qgismobileapp.qml

```diff
@@ -1832,10 +1853,14 @@ ApplicationWindow {
 
       onCancel: {
           if ( stateMachine.state === 'measure' && elevationProfileButton.elevationProfileActive ) {
-              elevationProfile.clear();
-              elevationProfile.refresh();
+              elevationProfile.clear()
+              elevationProfile.refresh()
           } else {
               if ( geometryRequested ) {
+                  if ( overlayFeatureFormDrawer.isAdding )
+                  {
```

Comment:
Sorry for being a pain, but in JS usually the opening `{` is on the same line as the if, look at the line before. I am ok to change it, but at least sticking to a single convention in the same file.

Reply:
Fair enough